src/components/postBox/PostUpdate.js

```diff
@@ -0,0 +1,81 @@
+/* import React from "react";
```

Comment:
Om vi ska använda denna, se till så att det är rätt kod och importeras/ exporteras rätt in i PostBox